JS_6/task2.2.js

```diff
@@ -0,0 +1,23 @@
+// 2. Написать функцию, которая будет принимать 1 аргумент (id отдела или название отдела и возвращать название предприятия, к которому относится).
+
+// Пример:
+// getEnterpriseName(4) // Предприятие 1
+// getEnterpriseName("Отдел маркетинга") // Предприятие 2
+
+import { enterprises } from "./task2.1.js";
+
+
+const getEnterpriseName = ((search) => {
+    for (const enterprise of enterprises) {
+        for (const department of enterprise.departments) {
+            if (department.id === search || department.name === search) {
+                return enterprise.name;
+            }
+        }
+    }
+    return `Предприятие не найдено`
```

Comment:
```suggestion
    throw new Error(`Предприятие не найдено`)
```
If function returns an object - you shall not return string in any case. Only throwing an Error is ok